backends/tc/ebpfCodeGen.cpp

```diff
@@ -965,16 +960,68 @@ void EBPFTablePNA::emitAction(EBPF::CodeBuilder *builder, cstring valueName,
                 builder->target->emitTraceMessage(builder, msgStr.c_str());
             }
         }
+        bool generateDefaultMissCode = false;
+        for (auto tbl : tcIR->tcPipeline->tableDefs) {
+            if (tbl->getTableName() == table->container->name.originalName) {
+                if (tbl->isTcMayOverride) {
+                    cstring defaultActionName = tbl->defaultMissAction->getActionName();
+                    auto actionNameStr = defaultActionName.c_str();
+                    for (long unsigned int i = 0; i < defaultActionName.size(); i++) {
+                        if (actionNameStr[i] == '/') {
+                            defaultActionName =
```

Comment:
will cstring's find/ findlast solve the purpose?

Reply:
Both these functions returns pointer to the character in the string, but not actual position, hence they cannot be used

Reply:
ref: test_backend.cpp
            fieldString = fieldString.substr(fieldString.find('.') - fieldString.begin() + 1);

Reply:
Updated